src/main/markdown/roadmap.md

```diff
@@ -0,0 +1,25 @@
+Roadmap
```

Comment:
Maybe expand the text a little further and just mention some very high level goals like catching up on Java language features up to 21, catching up on emulation and removing classic DevMode?

For the concrete releases then only link to their corresponding Github milestone. That would avoid duplicating milestone information/issues and you don't have the burden to keep it in sync in case planing changes.

Reply:
You are right. Using more high level goals will keep the amount of maintaining low.
Good point.

Reply:
I don't think we have any intention of removing legacy/classic dev mode - it has long since (2.8?) been discouraged/deprecated, but I'm not aware of anything that would necessitate removing it? Certainly there are things it can't do.

Reply:
ok. I'll remove the deprecated/removing stuff.

Reply:
I was more thinking along the lines of:

"GWT's roadmap is planned on Github using [Github Milestones](https://github.com/gwtproject/gwt/milestones). Milestones should cover the next two releases and because priorities can change please check milestones directly on Github for all the details.

As a high level goal next releases of GWT focus on bringing Java language features as well as JRE emulation up to Java 21. Additionally GWT explores support of some J2CL exclusive JsInterop features in GWT compiler."

That's it. No listing of releases at all.

Reply:
You are right. Sounds good. Just change 'Milestones should cover' into 'Milestones will cover' and chhanged 'next two releases' to 'next releases'.